caches/FileCache.php

```diff
@@ -92,7 +94,13 @@ public function setParameters(array $param){
 	*/
 	protected function getPath(){
 		if(is_null($this->path)) {
-			throw new \Exception('Call "setPath" first!');
+			throw new \Exception('Call "setScope" first!');
+		}
+
+		if(!is_dir($this->path)) {
+			if (mkdir($this->path, 0755, true) !== true) {
```

Comment:
One minor change, not necessarily related to this PR. The permission 0755 should be 0644 because we don't want execute permission in the cache folder, right?

```suggestion
			if (mkdir($this->path, 0644, true) !== true) {
```

Haven't tested it though...

Reply:
Sorry, after testing, I had to revert it to 0755. The execute bit is required to change into that folder in order to read/write files in there. I didn't think about that earlier.